src/apps/relay/ns_ioalib_engine_impl.c

```diff
@@ -3503,25 +3505,257 @@ const char *get_ioa_socket_ssl_method(ioa_socket_handle s) {
   return "no SSL";
 }
 
-void stun_report_binding(void *a, STUN_PROMETHEUS_METRIC_TYPE type) {
+/**
+ * @brief  turnserver: Get a uniqe session ID wrt. all sessions currently
+ *	managed by the associated turnserver.
+ *
+ * It is different wrt. the sessionID, because sessionID is a composit of the
+ * turnserverID and its session count and thus always in any scope (global or
+ * local) unique for the whole lifetime of the turnserver.
+ *
+ * However if sessionID would be used as parameter for timeseries metrics,
+ * this would let explode the number of timeseries (every sessionID creates a
+ * new one) and extremly stress the TDB.
+ *
+ * Using a recyclable ID instead reduces such timeseries to a more or less
+ * predictable level and improves reporting as well as data management time and
+ * server resources quiet a bit.
+ *
+ * The simple idea is: If a session gets closed, its ID gets returned into the
+ * related pool. Only if the pool has no ID, which is in there for the given
+ * duration, a new ID gets allocated/returned.
+ *
+ * @param ss	The sessions, which needs to get a recyclable ID.
+ */
+#ifdef POOL_TRACE
+#define _TRACE TURN_LOG_FUNC
+#else
+// dirty talk - is for developers, only
```

Comment:
I'm not really understanding this comment

Reply:
Would be `log trashing` better?